src/app/Caches/DataDistribution.tsx

```diff
@@ -0,0 +1,148 @@
+import React, { useState } from 'react';
+import {
+    Card,
+    CardBody,
+    CardTitle,
+    Level,
+    LevelItem,
+    Select,
+    SelectOption,
+    SelectVariant,
+    Spinner,
+} from '@patternfly/react-core';
+import { Chart, ChartBar, ChartGroup, ChartVoronoiContainer } from '@patternfly/react-charts';
+import { TableErrorState } from '@app/Common/TableErrorState';
+import { useTranslation } from 'react-i18next';
+import { useDataDistribution } from '@app/services/dataDistributionHook';
+import { DataDistributionStatsOption } from "@services/infinispanRefData";
+import { PopoverHelp } from '@app/Common/PopoverHelp';
+
+const DataDistribution = (props: {
+    cacheName: string;
+}) => {
+    const { t } = useTranslation();
+    const brandname = t('brandname.brandname');
+
+    const [isOpenStatsOptions, setIsOpenStatsOptions] = useState<boolean>(false);
+    const [statsOption, setStatsOption] = useState<string>(DataDistributionStatsOption.Entries);
+
+    const { dataDistribution, loading, error } = useDataDistribution(props.cacheName);
+
+    const onSelectStatsOptions = (event, selection, isPlaceholder) => {
+        if(selection === t('caches.cache-metrics.data-distribution-option-entries'))
+            setStatsOption(DataDistributionStatsOption.Entries);
+        else if(selection === t('caches.cache-metrics.data-distribution-option-memory'))
+            setStatsOption(DataDistributionStatsOption.Memories);
```

Comment:
we are passing the hardcoded enum here instead of the label. this should be done the other way around

Reply:
For  enum: 
```
export enum DataDistributionStatsOption {
  Entries = 'Entries',
  Memories = 'Memorie entries',
}
```

For translation:
```      
"data-distribution-option-entries": "Total number of entries",
"data-distribution-option-memory": "Total memory used",
```
The values in both are different, can't we use "Total number of entries" in the enum instead of "Entries"?

Reply:
I will check the PR and I tell you